config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -598,6 +598,25 @@ postsubmits:
         resources:
           requests:
             memory: "1Gi"
+  - name: post-test-infra-upload-triage
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^triage/Makefile$|^triage/[^/]+(\.html|\.js|.css)$'
+    decorate: true
+    spec:
+      serviceAccountName: pusher
```

Comment:
granted permission for k8s-gubernator bucket to pusher@k8s-prow.iam.gserviceaccount.com